patches/@tldraw+tldraw++@tldraw+editor+2.0.0-canary.ffda4cfb.patch

```diff
@@ -0,0 +1,89 @@
+diff --git a/node_modules/@tldraw/tldraw/node_modules/@tldraw/editor/dist/cjs/index.d.ts b/node_modules/@tldraw/tldraw/node_modules/@tldraw/editor/dist/cjs/index.d.ts
```

Comment:
we are hacking tldraw for our purposes. Long term, we want to push for these changes upstream